### PR TITLE
Updated teams page

### PR DESCRIPTION
**Pull request by russjones**
Updated teams page to include mission statements for Core and Cloud teams.

* [Rendered View](https://github.com/gravitational/careers/blob/rjones/update-teams-mission/teams.md)

### CODE REVIEW

File: teams.md [hunk 1]
```diff
@@ -1,220 +1,280 @@
...
-### Cloud Reliability
+Deliver outstanding user experience for Cloud users and the Sales team.
 
-Our Cloud Reliability Team defines and builds a culture around production,
-ensuring scaling of our product offering against company/product growth.
+Recent projects:
 
-Some projects we're working on right now:
+* Add support for Upgrade Windows
+* Simplify the Teleport Downloads page
```
Comment (jimbishopp) [inline teams.md:217]:
```suggestion
* Simplify the Teleport Downloads page
* User journey tracking

```

File: teams.md [hunk 2]
```diff
@@ -1,220 +1,280 @@
...
+for customers and the sales team.
 
-As the applications team for Teleport Cloud, we manage the Teleport deployments
-for our cloud customers and provide a stable, easy-to-use product.
+Recent projects:
 
-The team intersects with almost all other departments in the company and offers
-a broad range of projects. For example:
+* Reduce tenant on-boarding time
+* Build reporting and analytics platform
```
Comment (jimbishopp) [inline teams.md:228]:
```suggestion
* Build reporting and analytics platform
* Container registry migration
* Kubernetes operator enhancements
* DB Schema migration tool
```

File: teams.md [hunk 3]
```diff
@@ -1,220 +1,280 @@
...
-  access our infrastructure backing Teleport Cloud.
-- Improving static analysis and automated tests.
-- Developing security conscious CI guidelines and tooling.
-- While we build up the Cloud Security team, tooling is heavily invested in
-  improving our security posture
-  audit trail for generated and published assets and ensuring we're using
-  encryption at rest in the right places.
+* Switch to GitHub Actions
+* Develop more sophisticated resource growth alerts
+* Automate manual processes
```
Comment (jimbishopp) [inline teams.md:240]:
```suggestion
* Automate manual processes
* Platform monitoring and alerting enhancements
* Development release pipeline for testing local builds on Cloud
```

File: teams.md [hunk 4]
```diff
@@ -1,220 +1,280 @@
...
+
+Build highly-available, reliable and secure, multi-region, low latency
+networking and orchestration layer. Track and improve noisy neighbors,
+bandwidth, compute and network utilization.
+
+Recent projects:
+
+* Reduce connection latency for geo-distributed infrastructure
+* Build next-generation Cloud platform infrastructure
+* Update disaster recovery and backups infrastructure
```
Comment (jimbishopp) [inline teams.md:252]:
```suggestion
* Multiple cloud provider account support
* Update disaster recovery and backups infrastructure
* Multi-region support for Kubernetes operator
```

File: teams.md [hunk 5]
```diff
@@ -1,220 +1,280 @@
...
 
-- Building and maintaining Teleport clusters that engineers use to securely
-  access our infrastructure backing Teleport Cloud.
-- Improving static analysis and automated tests.
-- Developing security conscious CI guidelines and tooling.
-- While we build up the Cloud Security team, tooling is heavily invested in
-  improving our security posture
-  audit trail for generated and published assets and ensuring we're using
-  encryption at rest in the right places.
+* Switch to GitHub Actions
```
Comment (jimbishopp) [inline teams.md:238]:
```suggestion
* CI/CD with GitHub Actions
* Publish metrics using custom GitHub Action
```

**Review by jimbishopp — APPROVED** [review]

File: teams.md [hunk 6]
```diff
@@ -1,220 +1,266 @@
...
+  [#12410](https://github.com/gravitational/teleport/pull/12410)
 
-The [Machine ID](https://goteleport.com/blog/machine-to-machine-access/) team is
-focused on bringing all the advantages and convenience that Teleport provides
-for human users to machine use cases.
+### Application Access
 
-The team is currently working on:
+The Application Access team works on providing access to web and console
+applications programs behind NATs and firewalls with security and compliance
```
Comment (zmb3) [inline teams.md:72]:
```suggestion
applications behind NATs and firewalls with security and compliance
```

**Review by zmb3 — APPROVED** [review]

File: teams.md [hunk 7]
```diff
@@ -1,220 +1,266 @@
...
 
-- Golang
-- Typescript & React
-- PostgreSQL
-- Terraform and Packer
-- Amazon Web Services
-- Kubernetes
-- Prometheus / Alertmanager / Loki internally and on Grafana Cloud
-- Drone.io
+* Golang
```
Comment (r0mant) [inline teams.md:187]:
```suggestion
* Go
```

File: teams.md [hunk 8]
```diff
@@ -1,220 +1,266 @@
...
+* Typescript & React
+* PostgreSQL
+* Terraform and Packer
+* Amazon Web Services (AWS)
+* Kubernetes
+* Prometheus/Alertmanager/Loki internally and on Grafana Cloud
```
Comment (r0mant) [inline teams.md:193]: I think we're migrating off of Grafana Cloud soon?
```suggestion
* Prometheus/Alertmanager/Loki/Grafana
```

**Review by r0mant — APPROVED** [review]

**Pull request merged**